### PR TITLE
Calibration and simple pi0 analysis added

### PR DESCRIPTION
**Pull request by peressounko**

### CODE REVIEW

**Comment by peressounko** [comment]
Hi @mikesas, could you please have a look at this PR?

**Review by mikesas — APPROVED** [review]

**Review by mikesas — APPROVED** [review]

**Review by mikesas — APPROVED** [review]

**Comment by peressounko** [comment]
Dear @ktf could you please have a look, who can approve PRs from PWGEM? Mike's approval seems does not work...

**Review by jgrosseo — APPROVED** [review]

**Comment by jgrosseo** [comment]
@peressounko Mike's approval is enough for PWGEM but you also changed a file in Common which needs an additional approval. See https://github.com/AliceO2Group/O2Physics/blob/master/CODEOWNERS

**Pull request merged**
